tests/integration/targets/podman_pod/tasks/root-pod.yml

```diff
@@ -115,9 +115,11 @@
           - "4444:4444/tcp"
           - "8888:19191/udp"
           - "127.0.0.1:7671:7676/udp"
+          - "10.126.0.1:7671:7676/udp"
```

Comment:
It fails with error `Error: error starting container 0fd706662950af98a7b24891f529aaa1dedf7947988773123e16f9037c2b1059: cannot listen on the UDP port: listen udp4 10.126.0.1:7671: bind: cannot assign requested address`
I think it's better to use localhost addresses, like:

```yaml
- "127.0.0.1:7671:7676/udp"
- "127.0.0.2:7671:7676/udp"
 ```